freemocap/gui/qt/widgets/active_recording_widget.py

```diff
@@ -57,8 +57,13 @@ def get_active_recording_info(self, return_path: bool = False) -> Union[Recordin
 
     def set_active_recording(
         self,
-        recording_folder_path: Union[str, Path],
+        recording_folder_path: Union[str, Path, None],
     ):
+        if recording_folder_path is None or recording_folder_path == "None":
+            logger.info("No recording folder path provided - clearing active recording")
```

Comment:
...and if we need a way to 'clear' it (do we?), then we can add a new method for `...clear_active_recording()` or soemthing

Reply:
We can add a `clear_active_recording()` method if we would rather be explicit, but just setting the active recording to None gives the behavior we want. This is how `.active_recording_info` is initialized, and the active recording info widget already displays it well.